clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -64,11 +64,11 @@ public class MockConsumer<K, V> implements Consumer<K, V> {
     private final Map<TopicPartition, OffsetAndMetadata> committed;
     private final Queue<Runnable> pollTasks;
     private final Set<TopicPartition> paused;
+    private final AtomicBoolean wakeup;
 
-    private Map<TopicPartition, List<ConsumerRecord<K, V>>> records;
+    private final Map<TopicPartition, List<ConsumerRecord<K, V>>> records;
```

Comment:
Let's add the empty line after this field to separate the final fields from the other ones.

Reply:
done